src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/ScoringCard.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import _ from 'lodash';
+import isNil from 'lodash';
```

Comment:
This is wrong - it's importing the lodash full build (all functions, normally called `_`) under the name `isNil`. It's still a function so it doesn't obviously break, but it's definitely wrong. I'm not sure if `import { isNil } from 'lodash';` is going to work either because that version is designed to be used as NodeJS modules, not `import` ed. I think the following is the best / most correct:

```suggestion
import { isNil } from 'lodash-es';
```
See https://stackoverflow.com/a/50656694

Though this also requires changing the installed version from `lodash` to `lodash-es`. This also requires a hack to work with Jest (via `moduleNameMapper` in `jest.config.js`) but that hack is already in place for this MFE.

Reply:
For single imports like this one, we can also write: `import last from 'lodash/last'`. I'm updating this PR to use this more explicit syntax. Let me know if you agree.